package lassewesth.internal.terrible;

import org.junit.Test;

import static org.junit.Assert.fail;

public class TerribleTest {
    @Test
    public void shouldYikes() throws Exception {
        fail("TODO: " + new Accident());
    }
}
